ethers-etherscan/src/contract.rs

```diff
@@ -331,17 +331,23 @@ impl Client {
         }
 
         let query = self.create_query("contract", "getabi", HashMap::from([("address", address)]));
-        let resp: Response<String> = self.get_json(&query).await?;
-        if resp.result.starts_with("Max rate limit reached") {
+        let resp: Response<Option<String>> = self.get_json(&query).await?;
+
+        let result = resp.result.unwrap_or("".to_string());
```

Comment:
null will now always be a serde error,

I'd rather have a new error variant for EmptyResult or something

Reply:
done in https://github.com/gakonst/ethers-rs/pull/2426/commits/94ad922d9651ef3c1ff9cb5fc2acb0f5a43f5389 — we now bubble up the error with an `EmptyResult` variant